packages/webapp/src/containers/Animals/Inventory/index.tsx

```diff
@@ -145,6 +147,27 @@ function AnimalInventory({
         ),
         sortable: false,
       },
+      {
+        id: isDesktop ? 'location' : null,
+        label: t('ANIMAL.ANIMAL_LOCATIONS').toLocaleUpperCase(),
+        format: (d: AnimalInventory) => (
+          <Cell
+            kind={CellKind.PLAIN}
```

Comment:
There is a cell kind of "ICON_TEXT" did it not work?

Reply:
I just tried it out. We would need to work around the text (the component requires `ICON_NAME`, and the location needs to be the text `Unknown` for undefined locations) and pass custom styles. I think it's better not to use the `Cell`.

Reply:
Yup up to you!

I think 

```
d.location ? <Cell kind={ICONTEXT} text iconName  ../> : <Cell kind={PLAIN} text /> 
```

Would also work to keep text styles same across cells but I am good with anything.

Reply:
The location text style is unique 😅

Reply:
Hmm 🤔 .  Yeah, it is unique.... Is it designed to be different than all other cells? It seems strange to me to have one cell have different size, color, and weight of text. 

The whole point of `<Cell />` was to enforce the same text and other styles -- its almost useless otherwise haha.

Reply:
I believe the difference is intentional. I don’t have a strong sense of design, but it doesn’t seem strange to me :)